aptos-move/framework/move-stdlib/sources/i128.move

```diff
@@ -0,0 +1,237 @@
+module std::i128 {
+    /// Arithmetic operation resulted in overflow (value outside the range [-2^127, 2^127 - 1])
+    const EOVERFLOW: u64 = 1;
+    /// Division by Zero is not allowed
+    const EDIVISION_BY_ZERO: u64 = 2;
+
+    /// min number that a I128 could represent = (1 followed by 127 0s) = 1 << 127
+    const BITS_MIN_I128: u128 = 0x80000000000000000000000000000000;
+
+    /// max number that a I128 could represent = (0 followed by 127 1s) = (1 << 127) - 1
+    const BITS_MAX_I128: u128 = 0x7fffffffffffffffffffffffffffffff;
+
+    /// (1 << 128) - 1
+    const MAX_U128: u128 = 0xffffffffffffffffffffffffffffffff;
+
+    /// 1 << 128
+    const TWO_POW_128: u256 = 0x100000000000000000000000000000000;
+
+    const LT: u8 = 0;
+    const EQ: u8 = 1;
+    const GT: u8 = 2;
+
+    struct I128 has copy, drop, store {
```

Comment:
Comment here: `/// Implementation of the `i128` primitive type. Do not use this type direction, instead use `i128`.